components/director/internal/domain/bundle/fixtures_test.go

```diff
@@ -173,7 +173,10 @@ const (
 	tenantID         = "b91b59f7-2563-40b2-aba9-fef726037aa3"
 	externalTenantID = "eeeeeeeee-eeee-eeee-eeee-eeeeeeeeeeee"
 	ordID            = "com.compass.v1"
+	localTenantID    = "localTenantID"
 	correlationIDs   = `["id1", "id2"]`
+	version          = "v1.0.5"
```

Comment:
This is not a valid version according ORD spec!
Cannot start with letter and 0 that is not followed by dot!